cloudinit/sources/DataSourceNoCloud.py

```diff
@@ -48,6 +48,15 @@ def _get_devices(self, label):
                 if ('label "%s"' % label) in mscdlabel_out:
                     devlist.append('/dev/' + dev)
                     devlist.append('/dev/' + dev + 'a')  # NetBSD 7
+        elif util.is_OpenBSD():
+            out, _err = util.subp(['sysctl', '-n', 'hw.disknames'], rcs=[0])
+            devlist = []
+            for i in out.split(','):
```

Comment:
Would be good to use a more descriptive variable name than `i` here; I cannot infer what values it might take currently, and I don't have an OpenBSD system to check for myself.

Reply:
the code is now in cloudinit/util.py and has an unit-test.